src/test/java/com/string/ReverseWordsTest.java

```diff
@@ -0,0 +1,13 @@
+package com.string;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class ReveresWordsTest extends ReverseWords {
```

Comment:
This is not proper way to write tests. 

If you don't know how a test is to be written, please check [DecimalToAnyBaseTestjava](https://github.com/TheAlgorithms/Java/blob/Development/src/test/java/com/conversions/DecimalToAnyBaseTest.java). 

As you can see, it does not extend its source class. Instead it initializes its object with help of constructor. 

Please do needful.

Reply:
I removed the import line and inheritance from `ReverseWordsTest` and initialized an object of `ReverseWords` in it.